src/snowflake/snowpark/_internal/data_source/utils.py

```diff
@@ -53,6 +53,12 @@ class DBMS_TYPE(Enum):
     "sqlite3": SqliteDriver,
 }
 
+UDTF_PACKAGE_MAP = {
+    DBMS_TYPE.ORACLE_DB: ["oracledb"],
+    DBMS_TYPE.SQLITE_DB: ["sqlite3"],
+    DBMS_TYPE.SQL_SERVER_DB: ["pyodbc"],
```

Comment:
I'm thinking of another case, what if a dbmd has multiple driver support

e.g.

sqlserver supports 1 pyodbc + msodbcsql 2. pymssql
how can this design fit the multiple driver support story?

Reply:
if that is the case, we can put them all in the list,
for example, DBMS_TYPE.SQL_SERVER_DB: ["pyodbc", "msodbcsql", "pymssql"],

Reply:
let's think about this later when we actually need to support pymssql